tensorflow-mixin/dashboards/tensorflow-overview.libsonnet

```diff
@@ -0,0 +1,740 @@
+local g = (import 'grafana-builder/grafana.libsonnet');
+local grafana = (import 'grafonnet/grafana.libsonnet');
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local prometheus = grafana.prometheus;
+
+local dashboardUid = 'tensorflow-overview';
+local matcher = 'job=~"$job",instance=~"$instance",model_name=~"$model_name"';
+
+local promDatasourceName = 'prometheus_datasource';
+local lokiDatasourceName = 'loki_datasource';
+
+local promDatasource = {
+  uid: '${%s}' % promDatasourceName,
+};
+
+local lokiDatasource = {
+  uid: '${%s}' % lokiDatasourceName,
+};
+
+local servingOverviewRow = {
+  collapsed: false,
+  title: 'Overview',
+  type: 'row',
+};
+
+local modelRequestRatePanel = {
+  datasource: promDatasource,
+  description: 'Rate of requests over time for the selected model. Grouped by statuses.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 0,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'auto',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      unit: 'reqps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'single',
+      sort: 'none',
+    },
+  },
+  targets: [
+    prometheus.target(
+      'rate(:tensorflow:serving:request_count{job=~"$job",instance=~"$instance",model_name=~"$model_name"}[$__rate_interval])',
+      datasource=promDatasource,
+    ),
```

Comment:
For all these prometheus queries, I'd recommend to add a legend format (e.g.: https://github.com/grafana/jsonnet-libs/blob/fd5379a1fba2d572fc314a0395dd61e7df335948/gitlab-mixin/dashboards/gitlab-overview.libsonnet#L225) - This removes a lot of irrelevant or duplicated info for the dashboard panel.